geosnap/analyze/network.py

```diff
@@ -112,24 +154,49 @@ def isochrones_from_id(origin, network, threshold):
     return alpha
 
 
-def isochrones_from_gdf(origins, threshold, network, network_crs=4326, reindex=True):
+def isochrones_from_gdf(
+    origins,
+    threshold,
+    network,
+    network_crs=4326,
+    reindex=True,
+    algorithm="alpha",
+    ratio=0.2,
+    allow_holes=False,
+):
     """Create travel isochrones for several origins simultaneously
```

Comment:
Maybe we can also add "based on distance here" just to 100% clear about the impedance?

Reply:
here too we're just querying the network (where the impedance has already been defined) so we can't say its distance based

Reply:
May be other than distance as confirmed by @knaaptime